utils/docker/vcluster/README.md

```diff
@@ -24,18 +24,19 @@ At this time only emulated hardware storage are supported by this Docker platfor
 
 To build and deploy the Docker images, `docker` and optionally `docker-compose` shall be available.
 The docker host should have access to the [Docker Hub](https://hub.docker.com/) and
-[CentOS](https://www.centos.org/) official repositories.  Finally,
+[Rocky Linux](https://rockylinux.org/fr/) official repositories.  Finally,
```

Comment:
We should probably link to the english version of the webstite.
```suggestion
[Rocky Linux](https://rockylinux.org/) official repositories.  Finally,
```

Reply:
fixed with commit https://github.com/daos-stack/daos/pull/7640/commits/04f06f6804d065f368b0138af0d3dce0462a4484